arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -222,7 +263,7 @@
           "200": {

```

Comment:
Please add descriptions
get: "Lists all the storage accounts available under the given resource group."
resourceGroup: "The name of the resource group within the user's subscription."